### DISCUSSION TITLE
See what source lead to certain goal conversion

### DISCUSSION THREAD

**Discussion by xlinesoft** [plausible/analytics#2389] [Anything Else]
What is the easiest way to see all the sources that lead to a certain goal conversion? Clicking sources one by one is not fruitful.

**Comment by metmarkosaric** [comment]
hi @xlinesoft! you can click on any goal to filter the dashboard by it and then look at the Top Sources report which should then list all the sources of goal conversions only

**Comment by xlinesoft** [comment]
Thank you, it helps!
